src/Schema/Conveyor/MetadataSchemaConveyor.php

```diff
@@ -18,7 +20,7 @@
 use Yiisoft\Yii\Cycle\Exception\EmptyEntityPathsException;
 use Yiisoft\Yii\Cycle\Schema\SchemaConveyorInterface as Conveyor;
 
-class MetadataSchemaConveyor extends SchemaConveyor
+final class MetadataSchemaConveyor extends SchemaConveyor
```

Comment:
This class was intentionally made extendable.
Please revert all the class signature changes related to `final`